API.md

```diff
@@ -1156,22 +1156,26 @@ for more documentation.
         automobile and will use highways, `'walking'`, which avoids
         streets without sidewalks, and `'cycling'`, which prefers streets
         with bicycle lanes and lower speed limits for transportation via
-        bicycle. (optional, default `driving`)
+        bicycle. The `'mapbox/driving-traffic'` profile is not supported. (optional, default `driving`)
 -   `callback` **Function** called with (err, results)
 
 **Examples**
 
 ```javascript
 var mapboxClient = new MapboxClient('ACCESSTOKEN');
 // With options
-mapboxClient.getDistances([
-  [-95.4431142, 33.6875431],
-  [-95.0431142, 33.6875431],
-  [-95.0431142, 33.0875431],
-  [-95.0431142, 33.0175431],
-  [-95.4831142, 33.6875431]
-], {
-  profile: 'walking'
```

Comment:
Hmm, maybe the example should include the `profile` option, it's always nice to see explicit use in docs.